<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rezerwacja</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {

border-image: url(:/images/hotel_menu.jpg);

}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#page_0{

border-image: url(:/images/hotel_menu.jpg);

}
#page_1{

border-image: url(:/images/hotel_menu.jpg);

}
#page_2{

border-image: url(:/images/hotel_menu.jpg);

}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>191</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Klient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>Recepcja</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>Menadżer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>Sprzątaczka</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>181</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_1">
         <property name="text">
          <string>Rezerwacja</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Wynajem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Anuluj rezerwację</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Powrót</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>321</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nazwisko</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nazwisko">
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>254</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nr dowodu/telefonu lub e-mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dane">
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Pokój</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pokoj">
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>135</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Od</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="data1"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_16">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>280</y>
         <width>135</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Do</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="data2"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>350</y>
         <width>261</width>
         <height>71</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zarezerwuj</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Imię</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imie">
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>170</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>170</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rezerwacja</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Powrót</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
